drivers/gpio/gpio_utils.h

```diff
@@ -64,4 +65,39 @@ static inline void gpio_fire_callbacks(sys_slist_t *list,
 	}
 }
 
+static inline bool gpio_irq_flags_edge_both(int flags)
```

Comment:
How about `gpio_flags_int_edge_both`?

First, the flags are `GPIO_INT_foo` not `GPIO_IRQ_foo`, so using `irq` is confusing.

Second, we might want similar functionality for other parts of the flags, e.g. `gpio_flags_pull_up()`.  It'd be nice if functionally-related operations shared a common prefix.

Reply:
Done